sql/src/test/java/io/crate/integrationtests/TransportSQLActionTest.java

```diff
@@ -1808,6 +1809,55 @@ public void testNewColumn() throws Exception {
         ensureYellow();
         execute("insert into t (name, score) values ('Ford', 1.2)");
     }
+
+    @Test
+    public void testKilledJobLog() throws Exception {
+        execute("create table likes (" +
+                "   event_id string," +
+                "   item_id string" +
+                ") clustered into 1 shards with (number_of_replicas = 0)");
+        ensureYellow();
+        execute("SET GLOBAL stats.enabled = true");
```

Comment:
stats should be disabled in a safe way (if an exception/assertion error occurs) after the test run